hals/halext/HalExtiMXDma/iMX6SxDmaHw.h

```diff
@@ -195,7 +195,7 @@ SDMA_CHANNEL_CONFIG Imx6SxDmaReqToChannelConfig[] = {
     // (3) SDMA_REQ_HDMI_AUDIO
     {
         SDMA_UNSUPPORTED_REQUEST_ID,    // Script address
-        SDMA_REQ_IPU1,                  // DMA request ID
+        SDMA_REQ_HDMI_AUDIO,            // DMA request ID
```

Comment:
What's the impact of this change?
If it's important we'll need to rebuild the DMA HAL extension and get it re-signed.

Reply:
Looks like a typo in the original code.

Reply:
Channels are restricted by an ASSERT() in the code to have the same DMA request ID.